### ISSUE TITLE
Create a Dockerfile to create a docker image

### ISSUE THREAD

**Issue by Ayushpanditmoto** [Ayushpanditmoto/CollegeReboot#7]
Create a Dockerfile and
Dockerizing this web app
![Moby-logo](https://user-images.githubusercontent.com/31253617/206718453-5af36bb2-4a44-4532-aace-043d3c299e8e.png)

**Comment by lipwei1808** [comment]
Hi, is this issue still open? I would like to contribute

**Comment by Ayushpanditmoto** [comment]
@lipwei1808 yes its still open if you want to contribute then i will assigned you just confirm me ??by Y/N

**Comment by lipwei1808** [comment]
Yes I will try to take that

**Comment by lipwei1808** [comment]
@Ayushpanditmoto I have opened a PR please take a look
